### DISCUSSION TITLE
Can not download the radioss source files

### DISCUSSION THREAD

**Discussion by DRSH2001** [OpenRadioss/OpenRadioss#796] [Q&A]
When I try to download the source files of radioss by

git clone git@github.com:OpenRadioss/OpenRadioss.git

I get the message:

git@github.com: Permission denied (publickey)
fatal: Could not read from remote repository

How can I solve this problem? Thank you in advance for your help.

**Comment by PaulAltair** [comment]
Have you registered an SSH key etc. ? See steps in the Getting Started guide pdf from this page on the confluence OpenRadioss portal

https://openradioss.atlassian.net/wiki/spaces/OPENRADIOSS/overview
